docs/getting_started/configuration.rst

```diff
@@ -279,7 +279,20 @@ Otherwise, the controller won't be able to properly compensate for the end effec
 
     End effector variants supported by Trossen Robotics are provided in :class:`trossen_arm::StandardEndEffector`.
 
-.. note:: Currently, only rack-and-pinion end effectors are supported.
+.. tip::
+
+    New in version 1.8.3: the original rack-and-pinion end effector can be replaced with a custom end effector, e.g. :member:`trossen_arm::StandardEndEffector::no_gripper`.
+
+    If the original end effector is not detected at controller startup, a custom end effector is assumed until the next boot.
```

Comment:
Something like...

On startup, the controller checks if the gripper motor exists. If the motor is not detected, it assumes that the gripper assembly has been removed.